demo.cc

```diff
@@ -202,8 +217,13 @@ class Cruncher final {
   struct Consumer {
     const std::string& demo_id_;
     Box box_;
+
+    std::string current_image_ = "NO DATA YET\n";
```

Comment:
Hey, it's a first prototype.

By the time we launch a hosted demo (a couple days?), the could will ensure the first "real" image is generated before the user can request it.